modules/govuk/manifests/node/s_draft_frontend.pp

```diff
@@ -28,5 +28,7 @@
     'PLEK_SERVICE_LICENSIFY_URI': value => "https://licensify.${app_domain_internal}";
     'PLEK_SERVICE_MAPIT_URI': value => "https://mapit.${app_domain_internal}";
     'PLEK_SERVICE_SEARCH_URI': value => "https://search.${app_domain_internal}";
+    'PLEK_SERVICE_FEEDBACK_URI': value => "https://feedback.${app_domain_internal}";
+    'PLEK_SERVICE_INFO_FRONTEND_URI': value => "https://info-frontend.${app_domain_internal}";
```

Comment:
To have an effect on the routes in the Router API/Router, this change will need to affect the draft content store, and I'm not sure the draft content store is on this machine?